### PR TITLE
WRP-26318: Fix editable scroller a11y feature to not cut off panel's audio guidance

### PR DESCRIPTION
**Pull request by juwonjeong**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [x] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Fix the editable scroller a11y feature to not cut off panel's audio guidance
Audio guide statement updated according to the latest UX

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Remove announce() which is used in the focus handler in the editable scroller 
Fix audioGuide to be read with aria-label of iconItem when the iconItem is focused

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRP-26318

### CODE REVIEW

File: samples/sampler/stories/qa/Panels.js [hunk 1]
```diff
@@ -140,3 +153,285 @@ WithAutoFocusControl.parameters = {
...
+	return (
+		<Panels
+			index={panelIndex}
+			noCloseButton
+			onBack={backward}
+		>
+			<Panel>
+				<Header title="Panel with Editable Scroller" subtitle="Click pencil button to start edit mode" />
+				<div ref={divRef}>
+					<Button style={{marginLeft: '36px'}} onClick={forward} icon="edit" />
```
Comment (seunghoh) [inline samples/sampler/stories/qa/Panels.js:347]: Let's add `aria-label` to readout "Edit mode"

File: samples/sampler/stories/qa/IconItem.js [hunk 2]
```diff
@@ -260,12 +261,13 @@ export const EditableIcon = (args) => {
 								return (
 									<div key={item.index} className={classNames(css.itemWrapper, {[css.hidden]: item.hidden})} aria-label={`Icon ${item.index}`} data-index={item.index} style={{order: index + 1}} disabled={item.iconItemProps['disabled'] || item.hidden}>
 										<ContainerDivWithLeaveForConfig className={css.removeButtonContainer}>
-											{item.hidden ? null : <Button aria-label="Delete" className={css.removeButton} onClick={onClickRemoveButton} icon="trash" />}
-											{item.hidden ? null : <Button aria-label="Hide" className={css.removeButton} onClick={onClickHideButton} icon="minus" />}
-											{item.hidden ? <Button aria-label="Show" className={css.removeButton} onClick={onClickShowButton} icon="plus" /> : null}
+											{item.hidden ? null : <Button aria-label="Delete button" className={css.removeButton} onClick={onClickRemoveButton} icon="trash" />}
```
Comment (seunghoh) [inline samples/sampler/stories/qa/IconItem.js:264]: Are you sure? <Button> tag reads "button" so, if we change like this, TTS will reads "Delete button, button"

**Review by seunghoh — COMMENTED** [review]
These are not related to this PR but, let's follow up on the new ticket
- a11y doesn't tell when Delete/Hide completed
- a11y reads press left/right to move when selecting the hidden button, but the hidden button cannot move. Even, it doesn't need to enter the select mode.
- Long press on the deactivated icon a11y text should not reaout out the edit hint. It should something like "Icon 2 deactivated"

**Review by seunghoh — APPROVED** [review]

**Comment by seunghoh** [comment]
LGTM

**Review by 0x64 — APPROVED** [review]

**Pull request merged**